File: internal/provider/dependence/neo4jcsv/testdata/src/jssrc/w.js
let g = 0;
function run(s, i) {
  let x = 1; x += 2; x++; --x;
  s.f = 3; s.arr[i] = 4; g = x;
  let [a, b] = [x, x]; [a, b] = [b, a]; ({f: a} = s);
  let y = x = 7; s?.g; g ??= 1;
}

File: internal/provider/dependence/neo4jcsv/testdata/src/pythonsrc/w.py
G = 0
class S:
    def __init__(self): self.f = 0; self.arr = [0]*4
def run(s, i):
    global G
    x = 1; x += 2
    s.f = 3; s.arr[i] = 4; G = x
    a, b = x, x; a, b = b, a
    y = x = 7
    d = {}; d["k"] = 6
